julia/nfct/NFCT.jl

```diff
@@ -0,0 +1,345 @@
+module NFCT
+
+export NFCTplan,nfct_plan
+
+ending = ".so"
+
+if Sys.iswindows()
+	ending = ".dll"
+elseif Sys.isapple()
+	ending = ".dylib"
+end
+
+# path to .so file
+const lib_path = string( @__DIR__, "/libnfctjulia", ending )
+
+# NFCT flags
+PRE_PHI_HUT = UInt32(1)<<0
+FG_PSI = UInt32(1)<<1
+PRE_LIN_PSI = UInt32(1)<<2 #
+PRE_FG_PSI = UInt32(1)<<3 #
+PRE_PSI = UInt32(1)<<4 #
+PRE_FULL_PSI = UInt32(1)<<5 #
+MALLOC_X = UInt32(1)<<6
+MALLOC_F_HAT = UInt32(1)<<7
+MALLOC_F = UInt32(1)<<8
+FFT_OUT_OF_PLACE = UInt32(1)<<9
+FFTW_INIT = UInt32(1)<<10
+NFCT_SORT_NODES = UInt32(1)<<11
+NFCT_OMP_BLOCKWISE_ADJOINT = UInt32(1)<<12
+PRE_ONE_PSI = (PRE_LIN_PSI| PRE_FG_PSI| PRE_PSI| PRE_FULL_PSI)
+
+# FFTW flags
+FFTW_MEASURE = UInt32(0)
+FFTW_DESTROY_INPUT = UInt32(1)<<0
+FFTW_UNALIGNED = UInt32(1)<<1
+FFTW_CONSERVE_MEMORY = UInt32(1)<<2
+FFTW_EXHAUSTIVE = UInt32(1)<<3
+FFTW_PRESERVE_INPUT = UInt32(1)<<4
+FFTW_PATIENT = UInt32(1)<<5
+FFTW_ESTIMATE = UInt32(1)<<6
+FFTW_WISDOM_ONLY = UInt32(1)<<21
```

Comment:
Does it make sense to use this flag (or some others like FFTW_CONSERVE_MEMORY , FFTW_PRESERVE_INPUT)